resources/lang/en/admin/hardware/form.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 return array(
+    'assets'            => 'Assets',
```

Comment:
Please remove this, and use general.assets instead. Otherwise translators have to translate the same word many times.